polylogue/drive.py

```
@@ -73,91 +155,104 @@ def get_drive_service(credentials_path: Path, verbose: bool = False):
+                    creds = _run_console_flow(flow, verbose=verbose)
             except Exception:
-                # fallback to console
-                creds = flow.run_console()
+                creds = _run_console_flow(flow, verbose=verbose)
             try:
                 token_path.parent.mkdir(parents=True, exist_ok=True)
-                with open(token_path, "w") as f:
+                with open(token_path, "w", encoding="utf-8") as f:
                     f.write(creds.to_json())
             except Exception:
                 pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)